### PR TITLE
Firebase integration

### PR DESCRIPTION
**Pull request by ReactwithHemanth**
Adding screens and custom modules and their relative libraries

### CODE REVIEW

File: src/Screens/EntryScreens/SignUpMethods.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                styles.Button,
+                {
+                  flexDirection: 'row',
+                  backgroundColor: colorArray[index],
+                },
+              ]}>
+              <View style={{marginHorizontal: 25}}>{item.icon}</View>
+              <Text
+                style={{
+                  textAlign: 'center',
```
Comment (vinay-sayone) [inline src/Screens/EntryScreens/SignUpMethods.js:58]: Remove inline styles

File: src/Screens/Home.js [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
 import {useAUth} from '../hooks/useAuth';
+import styles from '../Styles/styles';
 
 const Home = () => {
   const user = useAUth();
   return (
-    <View style={style.container}>
-      <Text style={{fontSize: 20, marginLeft: 30}}>Welcome {user.email}</Text>
+    <View style={styles.Container}>
+      <Text style={{fontSize: 20, marginLeft: 30}}>Welcome {user?.email}</Text>
```
Comment (vinay-sayone) [inline src/Screens/Home.js:10]: Remove inline styles

**Review by vinay-sayone — CHANGES_REQUESTED** [review]
Please check comments added.

File: src/Screens/Home.js (hunk 2 above)
Comment (ReactwithHemanth) [reply src/Screens/Home.js:10]: Removed Inline styles from Home.js

**Review by vinay-sayone — APPROVED** [review]
Approved

File: src/Screens/EntryScreens/SignUpMethods.js (hunk 1 above)
Comment (ReactwithHemanth) [reply src/Screens/EntryScreens/SignUpMethods.js:58]: Applied Changes

**Pull request merged**